app/javascript/src/modules/organisations.js

```diff
@@ -173,21 +93,14 @@ function hideOrganisationItems () {
   hide(document.querySelector('.no-organisation-items'))
 }
 
-function disableBackButton () {
-  window.history.pushState(null, document.title, window.location.href)
-  window.addEventListener('popstate', function (event) {
-    window.history.pushState(null, document.title, window.location.href)
-  })
-}
-
-if (window.location.href.includes('organisations_types')) {
-  disableBackButton()
+function deselectPreviousOrganisationItem () {
+  const selected = document.querySelector('input:checked')
```

Comment:
Would it be ideal to be more precise of the element we are trying to query, in this instance the class name?

Reply:
example? This was taken from proceedings and is selecting the input that is checked I guess of hich there should only ever be on 🤷 .